### DISCUSSION TITLE
Save as exemple ...

### DISCUSSION THREAD

**Discussion by GhosTVip3R** [usebruno/bruno#2697] [Ideas]
Hey there, Bruno API developers !
Exciting news ! We've just discovered a new feature that's going to make your lives a whole lot easier. It's called 'Save as Example' and it's a game-changer !

With this nifty tool, you can save your responses as examples. Ever found yourself scratching your head, trying to remember what a successful response looks like? Well, now you can save it and refer back to it whenever you need. 
No more guessing games or digging through old emails !

Just think of it as your personal response library. You can organize your examples by category, add notes, and even share them with your team. It's like having a cheat sheet for all your API requests.

So, what are you waiting for ?
Integrate 'Save as Example' into Bruno and start saving time, energy, and sanity.
Your future self will thank you !

**Comment by eifcgn24** [comment]
How do you integrate it?
